.github/workflows/fingertips-workflow.yml

```
@@ -272,6 +278,7 @@ jobs:
     uses: ./.github/workflows/e2e-and-api-tests.yml
     with:
       playwright-mode: docker
+      update_screenshots: ${{ inputs.update_screenshots || false }}
```

```suggestion
      update_screenshots: ${{ inputs.update_screenshots }}
```

Any issue with this? It defaults to false right?

---

i guess i just wanted to make sure that if this is null or undefined for some reason then the fallback position is false 

maybe i am just being paranoid

---

Yup, I think the default value covers us in this case.